### PR TITLE
Adds InputPickerRow capability back in to framework

### PR DESCRIPTION
**Pull request by simonmitchell**

### CODE REVIEW

File: ThunderTable/InputPickerViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	
+    override open func becomeFirstResponder() -> Bool {
+        return textField.becomeFirstResponder()
+    }
+    
+    override open func resignFirstResponder() -> Bool {
+        return textField.resignFirstResponder()
+    }
+	
+	open override func awakeFromNib() {
```
Comment (ryanbourneuk) [inline ThunderTable/InputPickerViewCell.swift]: Indentation seems a bit off in this file.

File: ThunderTable/Extensions.swift [hunk 2]
```diff
@@ -27,6 +27,48 @@ extension Array : Section {
     }
 }
 
+extension String: PickerRowDisplayable {
+	
+	public var rowTitle: String {
+		get {
+			return self
+		}
+		set {
```
Comment (ryanbourneuk) [inline ThunderTable/Extensions.swift]: If there's nothing within the `set` here, does it need to have an empty block? I can't remember if they're required or not :)

Comment (MattCheetham) [reply ThunderTable/Extensions.swift]: Yeah if you don't add the set {} it doesn't get recognised. Strange Swift limitation.

Comment (MattCheetham) [reply ThunderTable/Extensions.swift]: Should be on one line though

File: ThunderTable/InputPickerRow.swift [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+		return InputPickerViewCell.self
+	}
+	
+	private var cellTextField: UITextField?
+		
+	open override func configure(cell: UITableViewCell, at indexPath: IndexPath, in tableViewController: TableViewController) {
+		
+		guard let pickerCell = cell as? InputPickerViewCell else { return }
+		
+		super.configure(cell: cell, at: indexPath, in: tableViewController)
```
Comment (MattCheetham) [inline ThunderTable/InputPickerRow.swift]: Would you not want to call super before the guard? Seems strange to just return without calling super if the cell isn't our subclass type.

File: ThunderTable/InputPickerViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+//  InputTextFieldViewCell.swift
+//  ThunderTable
+//
+//  Created by Simon Mitchell on 15/09/2016.
+//  Copyright © 2016 3SidedCube. All rights reserved.
+//
+
+import UIKit
+
+open class InputPickerViewCell: TableViewCell {
```
Comment (MattCheetham) [inline ThunderTable/InputPickerViewCell.swift]: Documentation?

**Review by MattCheetham — CHANGES_REQUESTED** [review]

**Comment by simonmitchell** [comment]
@MattCheetham @ryanbourneuk done :)

**Review by MattCheetham — APPROVED** [review]

**Pull request merged**